articles/openshift/oshift-how-apply-security-best-practices.md

```diff
@@ -0,0 +1,70 @@
+---
+title: How to apply security best practices in OpenShift
```

Comment:
As there aren't really step-by-step instructions in this article, I'd make it a reference article rather than a how to.

So change the title to: Security best practices for UKCloud for Openshift